templates/plugins/katello_httpd.erb

```diff
@@ -0,0 +1,25 @@
+#
+# WARNING: THIS CONFIGURATION WAS GENERATED BY KATELLO-CONFIGURE TOOL,
+# CHANGES WILL LIKELY BE OVERWRITTEN.
+#
+
+SSLOptions +StdEnvVars +ExportCertData +FakeBasicAuth
+
+Alias /pub /var/www/html/pub
+<Location /pub>
+Options +FollowSymLinks +Indexes
+</Location>
+
+<LocationMatch /rhsm|/subscription|/katello/api>
```

Comment:
Is rhsm and subscription part of the plugin? I thought that was a separate process which should configure its own apache.